src/stratis_cli/_actions/_data.py

```diff
@@ -86,6 +86,33 @@
 <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="const"/>
 </property>
 </interface>
+""",
```

Comment:
Should get rid of the XML for the original Manager interface as well, since the code just builds the method for Manager.r1.

Reply:
I'll test it and see if that's possible. My guess is that this is doable as I think checking for interface consistency between stratis-cli and stratisd is done at the individual interface level, so if that's the case, I'll make that edit.

Reply:
Resolved.